pkg/looker.go

```diff
@@ -0,0 +1,183 @@
+package pkg
+
+import (
+	"concierge/config"
```

Comment:
run go fmt once.

Reply:
i did run it again.. no changes detected.